byu-feature-card/byu-feature-card-common.scss

```diff
@@ -43,8 +43,8 @@ $white: #FFFFFF;
 
 @mixin titleSlottedContent() {
   width: 100%;
-  height: 50px;
-  padding: 13px 13px;
+  min-height: 10px;
+  max-height: 50px;
```

Comment:
I think we should adjust this to taller for if they have 2 lines - don't we still need some padding here?

Reply:
To make it work, I had to take out the padding and switch it to a margin (see line 61 and 66). I will take out the max height.